webpack.common.js

```diff
@@ -0,0 +1,62 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const pathToIndexHtml = require.resolve('./src/index.html');
+
+module.exports = {
+	entry: {
+		app: './src/app.js',
+		article: './src/article/article.component.js'
+	},
+	plugins: [
+		new HtmlWebpackPlugin({title: 'Production'}),
+		new MiniCssExtractPlugin({
+			filename: "[name].css",
+			chunkFilename: "[id].css"
+		})
+	],
+	output: {
+		filename: '[name].bundle.js',
```

Comment:
Usually it's a good idea to add hash for production build. So, it'll be pretty straightforward to enable caching for all static files.